### PR TITLE
tests: Use new GetPhysicalDevice helpers

### PR DESCRIPTION
**Pull request by sjfricke**
should help address some failures in https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/4425

### CODE REVIEW

File: tests/positive/command.cpp [hunk 1]
```diff
@@ -602,8 +602,7 @@ TEST_F(VkPositiveLayerTest, DrawIndirectCountWithFeature) {
     auto features12 = LvlInitStruct<VkPhysicalDeviceVulkan12Features>();
     features12.drawIndirectCount = VK_TRUE;
     auto features2 = LvlInitStruct<VkPhysicalDeviceFeatures2>(&features12);
-
-    vk::GetPhysicalDeviceFeatures2(gpu(), &features2);
+    GetPhysicalDeviceFeatures2(features2);
```
Comment (ncesario-lunarg) [inline tests/positive/command.cpp:605]: No change needed, but just an FYI, this function is specialized such that if you pass in a non-`VkPhysicalDeviceFeatures2` struct, one will be created and returned with the struct you do pass in set to the pNext of the PDF2 struct. IOW, you can remove the previous line and just do `auto features2 = GetPhysicalDeviceFeatures2(features12);`.

**Review by ncesario-lunarg — APPROVED** [review]
LGTM, thanks!

**Comment by lunarpapillo** [comment]
FYI, the LunarG CI run will fail due to machine lockup.  I'm restarting.

**Comment by lunarpapillo** [comment]
And the last failure *also* seems to be a CI machine problem.  Restarting.

**Pull request merged**